sklearn/preprocessing/tests/test_data.py

```diff
@@ -1858,7 +1861,7 @@ def _check_transform_selected(X, X_expected, sel):
 
 
 def test_transform_selected():
-    X = [[3, 2, 1], [0, 1, 1]]
+    X = np.array([[3, 2, 1], [0, 1, 1]])
```

Comment:
why?

Reply:
left-over from a undo of previous changes, so not necessary I think, removed it.